frontend/src/components/mine/Incidents/MineIncidentTable.js

```diff
@@ -45,23 +72,25 @@ export class MineIncidentTable extends Component {
       followup_action: actions.find(
         (x) => x.mine_incident_followup_type_code === incident.followup_type_code
       ),
+      ...funcs,
```

Comment:
We've been moving away from spread operators (mainly for ...props but I don't see why other uses would be different) and are preferring to explicitly set properties unless they are required to be dynamic. Is there good reason not to pass in the two functions as separate parameters?

Reply:
thought it looked cleaner, but i suppose it hides things. changed.